src/aspire/utils/numeric/__init__.py

```diff
@@ -6,3 +6,12 @@
     from .numpy import Numpy as NumericClass
 
 xp = NumericClass()
+
+if config.common.normal_fft == 0:
+    from .pyfftw_fft import PyfftwFFT as FFTClass
+elif config.common.normal_fft == 1:
+    from .cupy_fft import CupyFFT as FFTClass
+else:
+    from .scipy_fft import ScipyFFT as FFTClass
+
+custom_fft = FFTClass()
```

Comment:
This is a bad name. Go with something very generic (`fft` etc.). The fact that it is custom should be a result of where it was imported from, and not part of its identity.

Reply:
Renamed to `fft` in commit ad35b7c.